### PR TITLE
8.0.0-alpha2 release notes 

### PR DESCRIPTION
**Pull request by jakelandis**
Release notes for 8.0.0-alpha2

### Preview

https://elasticsearch_77821.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/release-notes-8.0.0-alpha2.html

### CODE REVIEW

File: docs/reference/release-notes/8.0.0-alpha2.asciidoc [hunk 1]
```diff
@@ -3,4 +3,77 @@
...
+* Adding new PUT trained model vocabulary endpoint {es-pull}77387[#77387]
+* Creating new PUT model definition part API {es-pull}76987[#76987]
+
+Network::
+* Enable LZ4 transport compression by default {es-pull}76326[#76326] (issue: {es-issue}73497[#73497])
+
+Search::
+* [Rest Api Compatibility] Nested path and filter sort options {es-pull}76022[#76022] (issues: {es-issue}42809[#42809], {es-issue}51816[#51816])
+* [Rest Api Compatibility] `CommonTermsQuery` and `cutoff_frequency` param {es-pull}75896[#75896] (issues: {es-issue}42654[#42654], {es-issue}51816[#51816])
+* [Rest Api Compatibility] Allow first empty line for msearch {es-pull}75886[#75886] (issues: {es-issue}41011[#41011], {es-issue}51816[#51816])
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha2.asciidoc:58]:
```suggestion
* [REST API Compatibility] Nested path and filter sort options {es-pull}76022[#76022] (issues: {es-issue}42809[#42809], {es-issue}51816[#51816])
* [REST API Compatibility] `CommonTermsQuery` and `cutoff_frequency` parameter {es-pull}75896[#75896] (issues: {es-issue}42654[#42654], {es-issue}51816[#51816])
* [REST API Compatibility] Allow first empty line for `_msearch` {es-pull}75886[#75886] (issues: {es-issue}41011[#41011], {es-issue}51816[#51816])
```

File: docs/reference/release-notes/8.0.0-alpha2.asciidoc [hunk 2]
```diff
@@ -3,4 +3,77 @@
...
+
+
+Security::
+* Adding base `RestHandler` class for Enrollment APIs {es-pull}76564[#76564] (issue: {es-issue}76097[#76097])
+* Generate and store password hash for elastic user {es-pull}76276[#76276] (issue: {es-issue}75310[#75310])
+* Set elastic password and generate enrollment token {es-pull}75816[#75816] (issue: {es-issue}75310[#75310])
+* Enroll additional nodes to cluster {es-pull}77292[#77292]
+* Default hasher to PBKDF2_STRETCH on FIPS mode {es-pull}76274[#76274]
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha2.asciidoc:66]:
```suggestion
* Default hasher to `PBKDF2_STRETCH` on FIPS mode {es-pull}76274[#76274]
```

File: docs/reference/release-notes/8.0.0-alpha2.asciidoc [hunk 3]
```diff
@@ -3,4 +3,77 @@
...
+* [Rest Api Compatibility] Nested path and filter sort options {es-pull}76022[#76022] (issues: {es-issue}42809[#42809], {es-issue}51816[#51816])
+* [Rest Api Compatibility] `CommonTermsQuery` and `cutoff_frequency` param {es-pull}75896[#75896] (issues: {es-issue}42654[#42654], {es-issue}51816[#51816])
+* [Rest Api Compatibility] Allow first empty line for msearch {es-pull}75886[#75886] (issues: {es-issue}41011[#41011], {es-issue}51816[#51816])
+
+
+Security::
+* Adding base `RestHandler` class for Enrollment APIs {es-pull}76564[#76564] (issue: {es-issue}76097[#76097])
+* Generate and store password hash for elastic user {es-pull}76276[#76276] (issue: {es-issue}75310[#75310])
+* Set elastic password and generate enrollment token {es-pull}75816[#75816] (issue: {es-issue}75310[#75310])
+* Enroll additional nodes to cluster {es-pull}77292[#77292]
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha2.asciidoc:65]:
```suggestion
* Add `elasticsearch-enroll-node` tool {es-pull}77292[#77292]
```

File: docs/reference/release-notes/8.0.0-alpha2.asciidoc [hunk 4]
```diff
@@ -3,4 +3,77 @@
...
+
+
+[[enhancement-8.0.0-alpha2]]
+[float]
+=== Enhancements
+
+Authentication::
+* Autogenerate and print elastic pwd on startup {es-pull}77291[#77291]
+* Enroll Kibana API uses Service Accounts {es-pull}76370[#76370]
+* Introduces reset-kibana-system-user tool {es-pull}77322[#77322]
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha2.asciidoc:37]:
```suggestion
* Add `reset-kibana-system-user` tool {es-pull}77322[#77322]
```

File: docs/reference/release-notes/8.0.0-alpha2.asciidoc [hunk 5]
```diff
@@ -3,4 +3,77 @@
...
+* System indices treated as restricted indices {es-pull}74212[#74212] (issue: {es-issue}69298[#69298])
+
+License::
+* Set `xpack.security.enabled` to true for all licenses {es-pull}72300[#72300]
+
+Packaging::
+* Remove no-jdk distributions {es-pull}76896[#76896] (issue: {es-issue}65109[#65109])
+
+Security::
+* Remove dashboard only reserved role {es-pull}76507[#76507]
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha2.asciidoc:27]:
```suggestion
* Remove `kibana_dashboard_only_user` reserved role {es-pull}76507[#76507]
```

File: docs/reference/release-notes/8.0.0-alpha2.asciidoc [hunk 6]
```diff
@@ -3,4 +3,77 @@
...
+
+
+[[enhancement-8.0.0-alpha2]]
+[float]
+=== Enhancements
+
+Authentication::
+* Autogenerate and print elastic pwd on startup {es-pull}77291[#77291]
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha2.asciidoc:35]:
```suggestion
* Autogenerate and print elastic password on startup {es-pull}77291[#77291]
```

File: docs/reference/release-notes/8.0.0-alpha2.asciidoc [hunk 7]
```diff
@@ -3,4 +3,77 @@
...
+* Enroll Kibana API uses Service Accounts {es-pull}76370[#76370]
+* Introduces reset-kibana-system-user tool {es-pull}77322[#77322]
+
+ILM+SLM::
+* Allow for setting the total shards per node in the Allocate ILM action #76775 {es-pull}76794[#76794] (issue: {es-issue}76775[#76775])
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha2.asciidoc:40]:
```suggestion
* Allow for setting the total shards per node in the Allocate ILM action {es-pull}76794[#76794] (issue: {es-issue}76775[#76775])
```

**Review by jrodewig — APPROVED** [review]
LGTM aside from a few nits. Thanks @jakelandis!

File: docs/reference/release-notes/8.0.0-alpha2.asciidoc [hunk 8]
```diff
@@ -3,4 +3,77 @@
 
 coming::[8.0.0-alpha2]
```
Comment (lcawl) [inline docs/reference/release-notes/8.0.0-alpha2.asciidoc:4]: > coming::[8.0.0-alpha2]

If you can remove this "coming" flag in this PR too, that'll save us an extra PR.

**Review by lcawl — APPROVED** [review]
ML PRs LGTM, thanks!

File: docs/reference/release-notes/8.0.0-alpha2.asciidoc (hunk 8 above)
Comment (jakelandis) [reply docs/reference/release-notes/8.0.0-alpha2.asciidoc]: done https://github.com/jakelandis/elasticsearch/commit/1d1b0f612f5f3aacd04481501f4daceec161dcee

**Pull request merged**